docs/usage/shoot_status.md

```diff
@@ -50,30 +50,45 @@ For example, if a webhook has rules for `CREATE/UPDATE` Pods or Nodes and `failu
+
+In most cases, you can avoid this by simply excluding the `kube-system` namespace from your webhook:
+
 ```yaml
-namespaceSelector:
-  matchExpressions:
-  - key: gardener.cloud/purpose
-    operator: NotIn
-    values:
-    - kube-system
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+webhooks:
+  - name: my-webhook.example.com
+    namespaceSelector:
+      matchExpressions:
+      - key: gardener.cloud/purpose
+        operator: NotIn
+        values:
+          - kube-system
+    rules:
+      - operations: ["*"]
+        apiGroups: [""]
+        apiVersions: ["v1"]
+        resources: ["pods"]
+        scope: "Namespaced"
 ```
 
-If the Shoot still has webhooks with either `failurePolicy={Fail,nil}` or `failurePolicy=Ignore && timeoutSeconds>15` that act on [critical resources](https://github.com/gardener/gardener/blob/master/pkg/operation/botanist/matchers/matcher.go#L60) in the `kube-system` namespace, Gardener will set the `HibernationPossible` to `False` indicating, that the Shoot can probably not be woken up again after hibernation without manual intervention of the Gardener Operator.
-`gardener-apiserver` will prevent any Shoot with the `HibernationPossible` constraint set to `False` from being hibernated, that is via manual hibernation as well as scheduled hibernation.
-
-> By setting `.controllers.shootCare.webhookRemediatorEnabled=true` in the gardenlet configuration, the auto-remediation of webhooks not following the best practices can be turned on in the shoot clusters.
-> Concretely, missing `namespaceSelector`s or `objectSelector`s will be added and too high `timeoutSeconds` will be lowered.
-> In some cases, the `failurePolicy` will be set from `Fail` to `Ignore`.
-> Gardenlet will also add an annotation to make it visible to end-users that their webhook configurations were mutated and should be fixed by them in the first place.
-> Note that all of this is no perfect solution and just done on a best effort basis.
-> Only the owner of the webhook can know whether it indeed is problematic and configured correctly.
-> 
-> Webhooks labeled with `remediation.webhook.shoot.gardener.cloud/exclude=true` will be excluded from auto-remediation.
+However, some other resources (mostly without namespaces) might still trigger the remediator, namely:
+- endpoints
+- nodes
+- podsecuritypolicies
+- clusterroles
+- clusterrolebindings
+- customresourcedefinitions
+- apiservices
+- certificatesigningrequests
+- priorityclasses
```

Comment:
Does it really make sense to enlist this here? Probably this will go out-of-sync eventually. What about the link to the source code file?

Reply:
Yeah... I debated that in my head but at the same time some people are not able / willing to read the code. And since this is intended for shoot owners that might not know GO / how to read the code, should we expect them to have to go in the code to understand something like this? Also, this is probably very low maintenance. What's your opinion ?

Reply:
My concerns still apply, but I agree that it's probably low maintenance since the list is not changed frequently. The code is not difficult to read even for YAML engineers 😉 Anyways, I don't insist on changing it.